miopengemm/CMakeLists.txt

```diff
@@ -14,6 +15,11 @@ file(GLOB_RECURSE source_files src/*.cpp)
 
 add_library(miopengemm ${source_files})
 
+# RUNPATH is set only when ROCM_RPATH is defined in the ENV
+if( DEFINED ENV{ROCM_RPATH} )
+  set ( CMAKE_SHARED_LINKER_FLAGS " -Wl,--enable-new-dtags -Wl,--rpath,$ENV{ROCM_RPATH}" )
```

Comment:
This should not be added to the CMakeLists.txt, this should be added in the cmake toolchain definition.

Reply:
Removed from CMakeLists.txt file and added as a parameter to the cmake command.